core/client/fs/src/test/java/alluxio/client/file/cache/store/PageStoreTest.java

```diff
@@ -61,6 +64,23 @@ public void test() throws Exception {
     }
   }
 
+  @Test
+  public void getOffset() throws Exception {
```

Comment:
also test the case offset exceeding page len?

Reply:
Added.